dspace-api/src/main/java/org/dspace/statistics/util/LocationUtils.java

```diff
@@ -62,7 +62,7 @@ static public String getContinentCode(String countryCode)
         String continent = countryToContinent.getProperty(countryCode);
         if (null == continent)
         {
-            logger.error("Unknown country code " + countryCode);
+            logger.info("Unknown country code " + countryCode);
```

Comment:
The message was bothering me too, but I'm not sure about the side-effects, either, so I'll leave it out of this particular PR.